qp_shogun/shogun/shogun.py

```diff
@@ -123,8 +123,8 @@ def generate_shogun_redist_commands(profile_dir, temp_dir,
     return cmds, output
 
 
-def run_shogun_to_biom(in_fp, biom_in, out_dir, level, version):
-    if version == 'redist':
+def run_shogun_to_biom(in_fp, biom_in, out_dir, level, version='alignment'):
+    if version in ('redist', 'alignment'):
         output_fp = join(out_dir, 'otu_table.%s.%s.biom'
```

Comment:
the alignment profile won't have an associated level. I think we need a separate if statement here.

Reply:
Maybe even better to have a separate write function for the shogun alignment biom. What do you think?

Reply:
In the current implementation the "level" is "profile", [see](https://github.com/qiita-spots/qp-shogun/pull/48/files#diff-243e58d6b48087c4b5d8716fad0f921dR211); this will yield a biom named [otu_table.alignment.profile.biom](https://github.com/qiita-spots/qp-shogun/pull/48/files#diff-b11fdd7fcb4ab368a204ddd435472753R468).

Reply:
Hmmm... Feels a little bit hack-y, but I like the output nomenclature! I'm happy to merge as is.